app/services/booking_bug/appointments.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+module BookingBug
+  class Appointments < Base
+    MAX_UPDATE_DAYS = 7 # used to limit number of entries queried to determine if an update ahs occurred.
+
+    # rubocop:disable MethodLength, AbcSize
```

Comment:
I think we can definitely come up with a better approach, potential one that can re-use parts between the different importers.

I will start thinking about this as the next item of work as don't think trying to refactor before merging makes any sense.

If you have any ideas/plans on how this should be done, give me a shout.